_members/malte-jochum.md

```diff
@@ -2,9 +2,32 @@
 layout: member
 id: malte-jochum
 name: Malte Jochum
-lat: 46.953535
-long: 7.445375
-affiliation: Institute of Plant Sciences, University of Bern, Bern, Switzerland
+email: malte.jochum@idiv.de
+homepage: http://maltejochum.de/
+lat: 51.319850
+long: 12.395299
+affiliation: German Centre for Integrative Biodiversity Research (iDiv) Halle-Jena-Leipzig & Leipzig University
+github: MalteJochum
 ---
 
+Malte is a community ecologist based at iDiv in Leipzig, Germany.
 
+## Projects / publications
+Schneider, F.D., Güntsch, A., Penone, C., Fichtmueller, D., Manning, P., Gossner, M.M., Jochum, M., Ries, B.K., Ostrowski, A. & Simons, N.K. (2019) Towards an ecological trait ‐ data standard. Methods in Ecology & Evolution, 2019, 1–14.
+
+https://besjournals.onlinelibrary.wiley.com/doi/full/10.1111/2041-210X.13288
+
+## Existing openscience trait datasets
+Temperate and tropical macroinvertebrate body length, body width, body mass from a paper offering length-mass regressions for temperate and tropical invertebrates:
```

Comment:
Did you consider adding a dataset entry at https://opentraits.org/datasets via https://github.com/open-traits-network/open-traits-network.github.io/tree/master/_datasets?